src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -24,6 +24,7 @@
 from ...image_processor import VaeImageProcessor
 from ...loaders import FromCkptMixin, LoraLoaderMixin, TextualInversionLoaderMixin
 from ...models import AutoencoderKL, UNet2DConditionModel
+from onnxruntime.training.ortmodule import ORTModule
```

Comment:
We cannot do this. Onnxruntime is not a required dependency and we don't want to mix onnx and PyTorch

Reply:
@patrickvonplaten thanks, this PR is for illustration purposes. We are debating moving this fix into optimum, will close this once it is no longer needed.